src/GoogleSignin.android.js

```diff
@@ -107,7 +107,7 @@ class GoogleSignin {
   signIn() {
     return new Promise((resolve, reject) => {
       const sucessCb = DeviceEventEmitter.addListener('RNGoogleSignInSuccess', (user) => {
-        this._user = user;
+        this._user = Object.assign({},user);
```

Comment:
Can you do exactly like #180: `this._user = {...user};`?

And you have to do it for `currentUserAsync()` too.